packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1711,6 +1712,40 @@ QUnit.module('options changed', moduleSetup, () => {
     });
 });
 
+QUnit.module('focusOnDeletion', () => {
+    const items = ['item 1', 'item 2', 'item 3'];
+    QUnit.module('focusStateEnabled', function(assert) {
+        QUnit.test('list item should not be focus when deleting when focusStateEnabled = false (T1226030)', function(assert) {
+            const $element = $('#list').dxList({
+                items,
+                focusStateEnabled: false,
+                allowItemDeleting: true,
+                itemDeleteMode: 'static',
+            });
+            let $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            $listItems.eq(0).find(`.${STATIC_DELETE_BUTTON_CLASS}`).trigger('dxclick');
+            $listItems = $element.find(toSelector(LIST_ITEM_CLASS));
+
+            assert.notOk($listItems.eq(0).hasClass('dx-state-focused'), 'Next Item is not focused');
+        });
+
+        QUnit.test('list item should be focus when deleting when focusStateEnabled = true (T1226030)', function(assert) {
+            const $element = $('#list').dxList({
+                items,
+                allowItemDeleting: true,
```

Comment:
u should set `focusStateEnabled=true` here to make this test working on mobile devices because this option has "false" value be default on non-desktop:

Reply:
[b8bba63](https://github.com/DevExpress/DevExtreme/pull/27230/commits/b8bba636980d2478c625678291fb31d98286ed70)